modules/dataquery/jsx/react.filterBuilder.js

```diff
@@ -50,7 +50,9 @@ var FilterRule = React.createClass({
 				"lessThanEqual" : "<=",
 		    	"greaterThanEqual" : ">=",
 		    	"startsWith" : "startsWith",
-		    	"contains" : "contains"
+		    	"contains" : "contains",
+		    	"isNull" : "isNull",
+		    	"isNotNull" : "isNotNull"
```

Comment:
Leigh said that null and notNull was fine, you can double check with her and if not I'll change the labels